### ISSUE TITLE
Pinch to zoom

### ISSUE THREAD

**Issue by dchenk** [Hikari9/comfortable-swipe#65]
@Hikari9 thanks for this wonderful tool. It works great. I'm planning on addressing issue #60 but am wondering if you'd want to support pinching to zoom in or out. Seems like something this app should be able to handle well. What do you think?

**Comment by Hikari9** [comment]
Thanks for reaching out, @dchenk. #60 would be great!

Although for pinch zoom, I'm kind of opinionated on it. 

Several attempts have actually been made, namely #8, #29, #46. The main reason we're putting it on air is because it's inconsistent as of the moment. There's still no way to do "smooth pinch" in the OS that can keep up with accuracy of hardware (at least that I am aware of).

The only available way to execute global pinch is the non-fine-tuned Ctrl {+,-}. `libinput` is really accurate in detecting pinch, but the OS itself has only finite pinch. There's also such a thing as 3-finger and 4-finger pinch, which conflicts (sometimes) with the swipe.

Other libraries like fusuma and libinput-gestures are already doing it with keyboard commands, but I don't want to mimic them for the sake of pinch. Comfortable-swipe's swipe gesture was made differently compared to the abovementioned libraries after all ☺

TLDR; So if you're gonna help out in #60, that will be really great! Pinch is still under debate, no need to put it out there yet.

**Comment by Hikari9** [comment]
PS: You might have noticed the unconventional file structure and lack of CMAKE. If you're an expert in packaging, feel free to shake up the structure to your preference while working on #60 and I'll take a look at it 🚀
